src/main/java/net/masterthought/cucumber/Configuration.java

```diff
@@ -136,13 +138,31 @@ public String getProjectName() {
         return projectName;
     }
 
+    /**
+     * Sets directory suffix.
+     *
+     * @param directorySuffix directory suffix
+     */
+    public void setDirectorySuffix(String directorySuffix) {
+        this.directorySuffix = directorySuffix;
+    }
+
+    /**
+     * Returns directory suffix
+     *
+     * @return directory suffix
+     */
+    public String getDirectorySuffix() {
+        return StringUtils.defaultString(directorySuffix);
+    }
+
     /**
      * Gets directory where the attachments are stored.
      *
      * @return directory for attachment
      */
     public File getEmbeddingDirectory() {
-        return new File(getReportDirectory().getAbsolutePath(), ReportBuilder.BASE_DIRECTORY
+        return new File(getReportDirectory().getAbsolutePath(), ReportBuilder.BASE_DIRECTORY + this.getDirectorySuffix()
```

Comment:
if you look how currently BASE_DIRECTORY looks like then IMO its better to have separator _ or / between - otherwise it might look strange
https://github.com/damianszczepanik/cucumber-reporting/pull/919/files#diff-e17c4a370f9a9da4a1fb9017744ebba7L46

Reply:
Ok, I made the change now to use `_` as suffix separator.